homeassistant/components/binance/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "binance",
+  "name": "Binance",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/binance",
+  "requirements": [
+    "python-binance-async-ha==0.7.3.2.dev0"
```

Comment:
This dependency looks like an outdated fork of `python-binance`. What the reason to use a fork or what issue does the fork solve?

Reply:
Ah good one, thanks! That was because back in the day `python-binance` did not support async operations. I see this has been fixed so I changed the dependency. I'll also rerun my tests to see if everything still works.